api/imapsync/update

```diff
@@ -75,8 +76,11 @@ if ($cmd eq 'update') {
         $deletefolders = '--delete2folders';
     }
 
-    my $exclude = join ('|',@{$input->{'Exclude'}}) || '';
+    my @exclusion = @{$input->{'Exclude'}};
+    push (@exclusion , '^Trash') if $input->{'TrashSync'} eq 'disabled';
+    my $exclude = join ('|', @exclusion) || '';
     $exclude = '|'.$exclude  if ($exclude ne '');
```

Comment:
Same as above, I'd go with a string-append operation. It's just one more line...

Reply:
done